src/rest-server/src/routes/v2/user.js

```diff
@@ -65,16 +66,18 @@ if (authnConfig.authnMethod === 'basic') {
     /** Put /api/v2/users */
     .put(
       token.checkNotApplication,
+      token.checkAdmin,
       param.validate(userInputSchema.basicAdminUserUpdateInputSchema),
```

Comment:
change the order to keep consistent?

Reply:
> change the order to keep consistent?

`Auth check` should have a higher level of priority than `content check`. `checkAdmin` is independent of the input schema, so I think it should be placed before.

Reply:
> change the order to keep consistent?

Order changed to keep the original logic.